src/napari/utils/info.py

```diff
@@ -145,6 +146,24 @@ def get_plugin_list() -> str:
         return f'Failed to load plugin information: <br> {e}'
 
 
+def startup_script() -> str:
+    """Get information about the startup script if executed."""
+
+    from napari.utils._startup_script import startup_script_status_info
+
+    if startup_script_status_info is None:
+        return ''
```

Comment:
```suggestion
    if startup_script_status_info is None:
        from napari.settings import get_settings
        return textwrap.dedent(f"""
        <br>Startup script:</b>
        - script path: {Path(get_settings().application.startup_script).expanduser().resolve()}<br>
        """)
```
This makes `napari --info` work for me locally.
I think this is important for troubleshooting.

Reply:
Used slightly different code to not print info if the script is not set.